packages/components/src/molecules/QuantitySelector/QuantitySelector.tsx

```diff
@@ -139,6 +140,10 @@ const QuantitySelector = ({
         value={useUnitMultiplier ? multipliedUnit : quantity}
         onChange={changeInputValue}
         onBlur={validateBlur}
+        onInput={(event: React.FormEvent<HTMLInputElement>) => {
+          const input = event.currentTarget
+          input.value = input.value.replace(/\D/g, '')
+        }}
```

Comment:
can't we use just `type="number"` here 🤔

Reply:
Yeah, it could be... we would probably have the same result

Reply:
actually, I think it’s safer to use with this validation ✅

Reply:
It seems to work well, but by adding this prop, the input appears with these little arrows and there is also a behavior that occurs when the user copies text and pastes it into the input, the value goes to 0, unlike the other approach, by js, which maintains the value that was entered, however it blocks any non-numeric character.

![image](https://github.com/user-attachments/assets/79581a3b-7209-44ea-81cc-f67eab14a80f)

Reply:
I don't know if It's a good idea to use type number if it changes de layout of the input.

Reply:
@Emanuel-Ramos sorry, I think the previous changes has a better approach 👍  Let's undo the latest changes (using `type=number`)